libbeat/docs/monitoring/monitoring-beats.asciidoc

```diff
@@ -1,74 +1,38 @@
-//// resolve Beat names: beatname_uc and beatname_lc.
-//// Use the following include to pull this content into a doc file:
-//// include::../../libbeat/docs/monitoring/configuring.asciidoc[]
-//// Make sure this content appears below a level 2 heading.
-//////////////////////////////////////////////////////////////////////////
-
 [role="xpack"]
 [[monitoring]]
 = Monitoring {beatname_uc}
 
 [partintro]
 --
 
-NOTE: The {monitor-features} for {beatname_uc} require {es} {beat_monitoring_version} or later.
-
-The {stack} {monitor-features} enable you to easily monitor {beatname_uc} from {kib}. For more
-information, see
-{stack-ov}/xpack-monitoring.html[Monitoring the {stack}] and
-{kibana-ref}/beats-page.html[Beats Monitoring Metrics].
+You can use the {stack} {monitor-features} to gain insight into the health of
+{beatname_uc} agents running in your environment. 
 
-To configure {beatname_uc} to collect and send monitoring metrics:
+To monitor {beatname_uc}, make sure monitoring is enabled on your {es} cluster,
+then configure the method used to collect {beatname_uc} metrics. You
+ifndef::serverless[]
+can use one of following methods:
+endif::[]
+ifdef::serverless[]
+can use the following method:
+endif::[]
 
-. Create a user that has appropriate authority to send system-level monitoring
-data to {es}. For example, you can use the built-in +{beat_monitoring_user}+ user or
-assign the built-in +{beat_monitoring_user}+ role to another user. For more
-information, see
-{stack-ov}/setting-up-authentication.html[Setting Up User Authentication] and
-{stack-ov}/built-in-roles.html[Built-in Roles].
+//REVIEWERS: Not sure what we should say for functionbeat users, but I don't
+//think it makes sense to include the metricbeat monitoring section in the
+//functionbeat docs. 
 
-. Add the `monitoring` settings in the {beatname_uc} configuration file. If you
-configured the {es} output and want to send {beatname_uc} monitoring events to
-the same {es} cluster, specify the following minimal configuration:
-+
-["source","yml",subs="attributes"]
---------------------
-monitoring:
-  enabled: true
-  elasticsearch:
-    username: {beat_monitoring_user}
-    password: somepassword
---------------------
-+
-If you configured a different output, such as {ls} or you want to send {beatname_uc}
-monitoring events to a separate {es} cluster (referred to as the _monitoring cluster_),
-you must specify additional configuration options. For example:
-+
-["source","yml",subs="attributes"]
---------------------
-monitoring:
-  enabled: true
-  elasticsearch:
-    hosts: ["https://example.com:9200", "https://example2.com:9200"] <1>
-    username: {beat_monitoring_user}
-    password: somepassword
---------------------
-<1> This setting identifies the hosts and port numbers of {es} nodes
-that are part of the monitoring cluster.
+* <<monitoring-internal-collection,Internal collection>>
+ifndef::serverless[]
+* <<monitoring-metricbeat-collection, {metricbeat}>>
```

Comment:
Should the link label say "Metricbeat collection" instead of just "Metricbeat" to parallel the "Internal collection" link label above it?

Reply:
sure!